sp-files/contributing-myst.md

```diff
@@ -103,11 +103,51 @@ Such structure makes it easier to review contributions and simplifies porting fi
 
 <!-- TODO: Update with your suggestions or drop if excessive -->
 
-To improve contribution tracking, we use the developer certificate of origin ([DCO 1.1](https://developercertificate.org/)) and require a "sign-off" for any changes going into each branch.
+To improve contribution tracking, we use the developer certificate of origin
+([DCO 1.1](https://developercertificate.org/) and require signed commits (using
+the `-S` or `--sign-off` option) for all changes that go into the ACME project.
 
-The sign-off is a simple line at the end of the commit message certifying that you wrote it or have the right to commit it as an open-source contribution.
+```{code-block} none
+git commit -S -m "acme/component: updated life cycle diagram"
+```
+
+Signed commits will have a GPG, SSH, or S/MIME signature that is
+cryptographically verifiable, and will be marked with a "Verified" or
+"Partially verified" badge in GitHub. This sign off verifies that you made the 
+changes or have the right to commit it as an open-source contribution.
+
+To set up locally signed commits and tags, see [GitHub Docs - About commit
+signature verification](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
+
+````{tip}
+Configure your Git client to sign commits by default for any local repository by
+running:
+
+```{code-block} none
+git config --global commit.gpgsign true
+```
+
+See [GitHub Docs - Signing commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
+````
+
+If you made an unsigned commit and encounter the "Commits must have verified
+signatures" error when pushing your changes to remote:
+
+1. Amend the most recent commit, add a signature without changing the commit
```

Comment:
Hey @akcano , off meaning?

For MD you can use "1. " for numbered lists instead of "1." "2." so it's easier to edit and manage.

Same like "1." followed by "#." for RST files.

If that is what you meant.